src/reporting/ReportingToken.sol

```diff
@@ -0,0 +1,150 @@
+pragma solidity ^0.4.13;
+
+import 'ROOT/libraries/DelegationTarget.sol';
+import 'ROOT/libraries/Typed.sol';
+import 'ROOT/libraries/Initializable.sol';
+import 'ROOT/libraries/token/VariableSupplyToken.sol';
+import 'ROOT/reporting/Interfaces.sol';
+
+
+contract ReportingToken is DelegationTarget, Typed, Initializable, VariableSupplyToken, IReportingToken {
```

Comment:
Hmm, something must be up with your merge.  This sholudn't be showing as a new file.